src/ui/api/repository.go

```diff
@@ -324,7 +324,7 @@ func (ra *RepositoryAPI) Delete() {
 			return
 		}
 		if repository == nil {
-			ra.HandleNotFound(fmt.Sprintf("repository %s not found", repoName))
+			log.Warningf("repository %s not found", repoName)
```

Comment:
Please add more details, such as `after deleting tag, the repo is not found` so we differentiate the scenario between tag deletion and repo deletion.